src/Workleap.OpenApi.MSBuild.sln

```diff
@@ -17,6 +17,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "debug", "debug", "{07F66FC2
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebApiDebugger", "WebApiDebugger\WebApiDebugger.csproj", "{A4C90BE2-889F-46BB-8B4D-2219B1084695}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "system-tests", "system-tests", "{4DDE83BF-D190-4CC9-AD36-E9250DABB27D}"
```

Comment:
In other project I see in IDP we name this solution folder "tests" should we do the same here? The project is already about system test.